src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
For self-closing tags, it is interesting to add the "/" before the closing arrow ">".